backend/src/main/java/ca/bc/gov/hlth/mohums/controller/UsersController.java

```diff
@@ -149,23 +149,26 @@ public ResponseEntity<List<Object>> getUsers(
         return searchResults;
     }
 
+    private boolean clientNameAndIdAreValid(String clientName, String clientId) {
+        LinkedHashMap client = (LinkedHashMap) webClientService.getClient(clientId).getBody();
+        return client.get("name").equals(clientName);
```

Comment:
Somewhat unrelated, but it's a bit funny that we deal with both client ID _and_ name. I couldn't tell you why offhand.